components/serverless/internal/webhook/validating_webhook_test.go

```diff
@@ -147,7 +147,11 @@ func TestValidatingWebHook_Handle(t *testing.T) {
 								},
 								"spec": {
 									"source": {
-                                        "gitRepository": {}
+                                        "gitRepository": {
+                                            "auth": {
+												"type": "invalid"
+											}
```

Comment:
Is it expected that this line is not formatted?

Reply:
Tabs & spaces were mixed, now only tabs are used